community/make-contribution/sign-icla-guide.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Sign ICLA Guide
+sidebar_position: 05
+description: Sign ICLA Guide
+---
+
+## Apache Sign ICLA Guide {#apache-sign-icla-guide}
+
+Apache CLA short for Apache Contributor License Agreement, the purpose of this agreement is to clearly define the terms under which intellectual property has been contributed to the ASF and thereby allow us to defend the project should there be a legal dispute regarding the software at some future time.
+A signed ICLA is required to be on file before an individual is given commit rights to any ASF project.
+
+If you have already signed an Apache ICLA, no need to send an email. As you have already submitted an ICLA under your public name, no need to resubmit it. If you have never signed an ICLA, please indicate this in your email, and PMC can apply for this account directly.
+
+The process of submitting ICLA are:
+
+**1. First, click the link [Apache Licenses](https://www.apache.org/licenses/#clas) and find CLAs under the Contributor License Agreements section.**
+
+![](/img/icla/page_link.png)
+
+**2. At the top of the page, there are two download links choose ICLA(Individual Contributor License Agreement) and download it.**
+
+![](/img/icla/download.png)
+
+**3. Open the PDF, and fill in the following columns in English.**
+
+> To be noticed:
+>
+> - Leave a space between family name and last name, remember capitalize the first letter, please write family name at first. Leave the public name empty or add a name with more than a single word; this is not an id
+> - Include your complete postal address where you receive postal mail, not a conference room or corporate address
+> - If you sign with a gpg signature, be sure that the email in the public key matches the email on the form
+> - Preferred Apache id(s) is a required field for committers.
+
+![](/img/icla/write_info.png)
+
+**4. The signature at the bottom of the document: Apache only accept hand-written signature or gpg signature, here we suggest choosing hand-written way. Print the document and sign your name on the paper, or use electric pen.**
+
+![](/img/icla/sign.png)
+
+## Email Template 
+
+![](/img/icla/email.png)
+
+addressee
+
+```
+secretary@apache.org
+```
+
+template
+
+```
+Hello Apache,
+    I am willing donate the DevLake project to ASF. The attachment is my ICLA information. Github account: https://github.com/xxxx.
```

Comment:
The template doesn't seem to be correct.

Reply:
@klesh I asked my friends and based on the replies I changed the email template